src/style.css

```diff
@@ -0,0 +1,283 @@
+@import url("http://fonts.cdnfonts.com/css/countryside");
+
+:root {
+  --primary-color: #012138;
+  --secondary-color: #f9f9f9;
+  --gradient-color: #024d83;
+  --layout-max-width: 1350px;
+}
```

Comment:
Please before merging, make sure that [Basic structure pull request](https://github.com/RafaelEchart/RestaurantMenu_Capstone/pull/22) is merged, then update your file to not complicate the styles and new structure.